Sources/TuistLoader/Loaders/SwiftPackageManagerGraphLoader.swift

```diff
@@ -165,6 +167,22 @@ public final class SwiftPackageManagerGraphLoader: SwiftPackageManagerGraphLoadi
             externalProjects: externalProjects
         )
     }
+
+    private func validatePackageResolved(at path: AbsolutePath) {
+        let savedPackageResolvedPath = path.appending(components: [
+            Constants.SwiftPackageManager.packageBuildDirectoryName,
+            Constants.DerivedDirectory.name,
+            Constants.SwiftPackageManager.packageResolvedName,
+        ])
+        let savedData = try? fileHandler.readFile(savedPackageResolvedPath)
```

Comment:
```suggestion
        let savedData: Data?
        if fileHandler.exists(savedPackageResolvedPath) {
          savedData = nil
        } else {
          savedData = try fileHandler.readFile(savedPackageResolvedPath)
        }
```

Let's check if the file exists and don't swallow the error if it does.